grep.js

```diff
@@ -21,18 +19,16 @@ const pickTests = (filename, foundTests, cypressConfig) => {
 
   const fgrep = cypressConfig.env.fgrep
   const grep = cypressConfig.env.grep // assume string for now, not regexp
-  if (!fgrep && !grep) {
```

Comment:
This is unnecessary. If they are not defined, the next two ifs will be skipped and `foundTests` will be returned.